### PR TITLE
Added i18n, html-helpers, console command without progressbar

### PR DESCRIPTION
**Pull request by Lunberg88**
denyCallback,matchCallback

### CODE REVIEW

**Comment by Lunberg88** [comment]
Gone

File: backend/models/Person.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ *
+ * @property integer $id
+ * @property string $name
+ * @property integer $age
+ * @property string $male
+ * @property string $date
+ *
+ * @property Address[] $addresses
+ */
+class person extends \yii\db\ActiveRecord
```
Comment (chabanenk0) [inline backend/models/Person.php:18]: Person (first letter captialized)

File: console/controllers/MyController.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+use yii\helpers\ArrayHelper;
+
+class MyController extends Controller
+{
+ 
+   public function actionIndex()
+    {
+
+$z = 0;
+while($z < 3) {
```
Comment (chabanenk0) [inline console/controllers/MyController.php:18]: Please use progressbar helper for displaying progress
https://gist.github.com/artemrogov/84b7ff6f68c20ffc0b54caa54ca053d7

File: console/controllers/MyController.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+use yii\db\Command;
+use yii\helpers\ArrayHelper;
+
+class MyController extends Controller
+{
+ 
+   public function actionIndex()
+    {
+
+$z = 0;
```
Comment (chabanenk0) [inline console/controllers/MyController.php:17]: Please fix the codestyle. Here should be 4*2 spaces in the line $z = 0; 
in order to be inside the block. Use your IDE (phpstorm) for proper code formating

File: console/migrations/m161218_144901_create_person_table.php [hunk 4]
```diff
@@ -1,31 +0,0 @@
-<?php
-
```
Comment (chabanenk0) [inline console/migrations/m161218_144901_create_person_table.php:2]: why did you delete this file? If you wanted to show only the certain hometask files, use different branches and create the pull request between them

File: frontend/views/site/about.php [hunk 5]
```diff
@@ -2,15 +2,99 @@
...
 $this->params['breadcrumbs'][] = $this->title;
 ?>
+ <br>
 <div class="site-about">
     <h1><?= Html::encode($this->title) ?></h1>
+<br>
+<?= Carousel::widget ( [
+    'items' => [
+    [
+        'content' => '<img style="width:474px;height:296px" src="//nix-tips.ru/wp-content/uploads/2014/11/carousel003.jpg"/>',
```
Comment (chabanenk0) [inline frontend/views/site/about.php:33]: don't use inline styles, add some class and move the styles to css file
